sbnobj/Common/Reco/CorrectedOpFlashTiming.h

```diff
@@ -23,25 +23,12 @@ namespace sbn {
 
   /// @name Corrected OpFlash Times
   /// @{
-  float        OpFlashT0             { fDefault };                   ///< | OpFlash Time wrt RWM time | (ns)
-  float        UpstreamTime_lightonly             { fDefault };      ///< | Nu upstream wall time reconstructed using light only | (ns)
-  float        UpstreamTime_tpczcorr             { fDefault };       ///< | Nu upstream wall time reconstructed using light and Z from tpc vertex | (ns)
-  float        UpstreamTime_propcorr_tpczcorr    { fDefault };       ///< | Nu upstream wall time reconstructed using light propagation correction from tpc information and z correction from tpc vertex | (ns)
-  
-  /// @}
-
-  /// @name Data members related to the slice-flash match
-  /// @{
-  float        FMScore             { fDefault };                   ///< | OpFlash Time wrt RWM time | (ns)
-
+  float        OpFlashT0             { fDefault };                   ///< | OpFlash Time wrt RWM time | (us)
+  float        NuToFLight             { fDefault };      ///< | Nu ToF using light only | (us)
+  float        NuToFCharge             { fDefault };       ///< | Nu ToF Z from tpc vertex | (us)
```

Comment:
In the documentation, could you spell "time of flight" out? It may save some time to unaccustomed readers.
```suggestion
  float        NuToFLight             { fDefault };      ///< | Nu time of flight using light only | (us)
  float        NuToFCharge             { fDefault };       ///< | Nu time of flight using Z from TPC vertex | (us)
```